src/redux/actions/userActions.js

```diff
@@ -82,6 +85,10 @@ const setAuthorizationHeader = token => {
   axios.defaults.headers.common['Authorization'] = Token;
 };
 
+const setUserHandleHeader = handle => {
```

Comment:
You should add some /** docs */ explaining what this does.

Reply:
Documentation added.